### PR TITLE
Add Chronos 2.5.0.

### PR DESCRIPTION
**Pull request by gkleiman**

### CODE REVIEW

**Comment by ryadav88** [comment]
@gkleiman Can we review https://github.com/dcos/examples/tree/master/chronos/1.9 and see if this needs edits based on your latest PR?

**Comment by gkleiman** [comment]
If we merge this PR, then users will be able to choose between Chronos v2.5.0 and v3.0.1.

The tutorial is for Chronos v3.0.1, so it doesn't need to change unless we decide to recommend users to use Chronos v2.50 instead of v3.0.1.

File: repo/packages/C/chronos/5/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "packagingVersion": "4.0",
+  "name": "chronos",
+  "version": "2.5.0",
+  "scm": "https://github.com/mesos/chronos.git",
+  "maintainer": "support@mesosphere.io",
+  "description": "A fault tolerant job scheduler for Mesos which handles dependencies and ISO8601 based schedules.",
+  "framework": true,
+  "minDcosReleaseVersion": "1.9",
```
Comment (ryadav88) [inline repo/packages/C/chronos/5/package.json]: @gkleiman you only want this to be available in `minDcosReleaseVersion ` >= 1.9 ?

Comment (gkleiman) [inline repo/packages/C/chronos/5/package.json]: Yes, we tested Chronos on the 1.9 on a soak cluster, but not on 1.8.

**Pull request merged**